opensrp-chw-core/src/main/java/org/smartregister/chw/core/utils/ReportUtils.java

```diff
@@ -16,11 +17,11 @@
 public class ReportUtils {
     private static final String TAG = ReportUtils.class.getCanonicalName();
 
-
     public static void createReport(List<ReportHia2Indicator> hia2Indicators, Date month, String reportType) {
         try {
             String providerId = CoreChwApplication.getInstance().getContext().allSharedPreferences().fetchRegisteredANM();
-            String locationId = CoreChwApplication.getInstance().getContext().allSharedPreferences().getPreference(CoreConstants.CURRENT_LOCATION_ID);
+            String locationId = LocationHelper.getInstance().getDefaultLocation();
+            CoreChwApplication.getInstance().getContext().allSharedPreferences().getPreference(CoreConstants.CURRENT_LOCATION_ID);
```

Comment:
Not sure where this is used. Looks like it isn't assigned to any variable? :thinking:

Reply:
It's been assigned here  report.setLocationId(locationId);